v0.3/tutorials/quickstart.eve

```diff
@@ -7,117 +7,95 @@ title: Quickstart
+## Getting Eve Running
 
-## Adding records to Eve
+Before you start with this tutorial, please follow the [installation and usage instructions](/v0.3/install/), which will get you running Eve programs on your machine. In the eve-starter directory, you can launch the Program Switcher:
 
-Let's start with a block that adds a record to Eve (we'll show the code fence in this block, but elide them in later blocks):
+```
+npm start
+```
 
-``````
-~~~
-commit
-  [#greeting message: "hello world"]
-~~~
-``````
+You'll see a list of Eve programs in your browser; select quickstart.eve in the program switcher, and use your favorite editor to open quickstart.eve so we can edit the program.
 
-The record committed by this block is tagged `#greeting`, and has an attribute "message" with the value "hello world". Tags are attributes like any other, their special syntax notwithstanding. We encourage you to use tags to classify groups of related records.
+## The Basics
+Let's start with the most basic example of Eve. Put this whole block, with the set of backticks (```) included, into the quickstart.eve file, then refresh your browser page where it's running.
 
-## Finding records in Eve
 
-Blocks search for records, and then bind or commit new records. Let's search for the `#greeting` we just committed, and then display it on the screen:
+```
+commit
+  [#ui/div text: "My Quickstart App"]
+```
 
-~~~eve
-search
-  [#greeting message]
+We want our webapp to be laid out nicely with a logo, a navigation bar, and a content area. To do that, add this whole block into the quickstart file right after the other block you already wrote, and refresh the browser page again.
 
-bind
-  [#ui/text text: message]
-~~~
 
-Searches find records that matches the supplied pattern, in this case records tagged greeting with a message attribute. You use these results to create new records. In this block, we create a `#ui/text` record for every message that is found (the UI library looks for records tagged `#ui/text` and draws them as DOM elements in the browser). If no records match the search, the block has no effect. Try adding another `#greeting` record in the first block to see what happens when more than one record matches the search. 
+```
+commit
+  [#ui/div #logo class:"qs-logo" sort: 1]
+  [#ui/div #nav-bar class:"qs-nav" sort: 2]
+  [#ui/div #contents class:"qs-contents" sort: 3]
+```
 
-Variables with the same name are equivalent within a block; because they have the same name, the `message` in `[#greeting message]` and `[#ui/text text: message]` are equivalent.
+The attribute `class` gets added to each div, which already has a little premade CSS to help see what we're doing. The logo is present, but next we need to add some pages to our nav bar and give each page some content.
```

Comment:
Instead of passive voice "gets added to", I'd make it active. "We add a class attribute to each div... to help see what we're doing"